### ISSUE TITLE
Typechecking error on CSS Modules using built-in VS Code TypeScript

### ISSUE THREAD

**Issue by chris-tse** [preactjs/wmr#203]
When using the default VS Code TypeScript version, it throws an error on the imported styles with an error, e.g. in index.js: `Property 'about' does not exist on type 'string'.` See screenshot below:

![css-modules-error](https://user-images.githubusercontent.com/250450/100959834-bd6a9a00-34e4-11eb-810d-4a82cb75686e.png)

## Steps to reproduce

1. Create project 
    ```
    npm init wmr my-new-project
    ```
2. Open up a page e.g. `public/pages/about/index.js`

Expected outcome: No TS errors shown
Actual outcome: Error above is shown on the default generated project.

## Possible fixes / workarounds

I was able to fix this by doing changing the `*.css` and other styles module declarations from:
```
declare module '*.css' { const url: string; export default url; }
```
to something like 
```
// Referenced from the typescript plugin linked below
declare module '*.css' { const classes: { [key: string]: string }; export default classes; }
```

However, I don't know whether the export was typed as a string on purpose however.

I was also able to fix this by doing the following:
1. Install `typescript` as a dev dependency `npm i -D typescript`
2. Installing something like [typescript-plugin-css-modules](https://www.npmjs.com/package/typescript-plugin-css-modules#recommended-usage)
3. Open VS Code Command Palette > `TypeScript: Select TypeScript Version...` > `Use Workspace Version`

After that, it is then able to  discern that the imported `styles` variable is not just a plain string. 

Is there a recommended way to fix this?

**Comment by lukeed** [comment]
Hey, this should be fixed by #200 -- it just hasn't been published yet :)

**Comment by chris-tse** [comment]
Awesome, thanks!
